src/components/LinearProgressBar/LinearProgressBar.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
+import {linearProgressBarDriverFactory} from './LinearProgressBar.driver';
+import {LinearProgressBar, LinearProgressBarProps} from './LinearProgressBar';
+import { linearProgressBarTestkitFactory } from '../../testkit';
+import {linearProgressBarTestkitFactory as enzymeLinearProgressBarTestkitFactory} from '../../testkit/enzyme';
+import { runTestkitExistsSuite } from '../../common/testkitTests';
+
+describe('LinearProgressBar', () => {
+
+    const createDriver = createDriverFactory(linearProgressBarDriverFactory);
+
+    const defaultProps = {
+        value: 40
+    }
+   
+    it('should exist', () => {
```

Comment:
Redundant test, because it is being tested with `runTestkitExistsSuite`. Please remove it